addons/mail/static/src/components/follower_subtype/follower_subtype.js

```diff
@@ -48,6 +55,7 @@ Object.assign(FollowerSubtype, {
     props: {
         followerLocalId: String,
         followerSubtypeLocalId: String,
```

Comment:
Those were identifying no? Should no longer be necessary and use the new relation instead.

Reply:
I make the changes in other following PRs:
https://github.com/odoo/odoo/pull/87014
https://github.com/odoo/odoo/pull/87013

It's not done in this PR to keep PR as small as possible while keeping working code.